tests/conftest.py

```diff
@@ -1544,6 +1544,22 @@ def collect_db_dump(request, duthosts):
     yield
     collect_db_dump_on_duts(request, duthosts)
 
+@pytest.fixture(autouse=True)
```

Comment:
It would be better to use scope 'module'.

Reply:
Yes, that would be better